docs/guides/rendering.md

```diff
@@ -1,445 +1,350 @@
+
+This effort is tracked by [issue #4795](https://github.com/napari/napari/issues/4795).
+It is partially complete as an experimental setting that should at least work for image-like layers.
+To enable the experimental setting, change it in napari's settings or preferences,
+or set `NAPARI_ASYNC=1` as an environment variable before running napari.
+
+The key code changes push all slicing (including synchronous slicing) through a dedicated controller [`_LayerSlicer`](https://github.com/napari/napari/blob/b3a8dd22895c913d8183735f52b9d1d71c963d7f/napari/components/_layer_slicer.py#L80),
+and define all the layer-specific slicing logic in dedicated callable classes (e.g. [`_ImageSliceRequest`](https://github.com/napari/napari/blob/b3a8dd22895c913d8183735f52b9d1d71c963d7f/napari/layers/image/_slice.py#L154)).
+An instance of one of these callables captures all the state needed to perform slicing,
+so that it can be executed asynchronously on another thread without needing to guard competing access to that state with locks.
+
+Unfortunately, these new additions make following the old synchronous slicing code paths more complicated.
+But eventually we hope to mostly remove those complications and make both synchronous and asynchronous slicing consistent and easy enough to follow.
+
+#### Future
+
+The current experimental asynchronous slicing approach is limited.
+While it prevents napari from blocking the main thread, fetching and rendering the data in view can still be slow.
+
+Most large image viewers improve on this experience with by progressively fetching and rendering chunks or tiles of data.
+This allows some data to be presented quickly rather than waiting for everything in view,
+which often results in a much better user experience when fetching the data is slow.
+
+These efforts across all layers are generally tracked by [issue #5942](https://github.com/napari/napari/issues/5942).
+Currently, the most focus is on the image layer in [issue #5561](https://github.com/napari/napari/issues/5561),
+with lots of progress towards that in [PR #6043](https://github.com/napari/napari/pull/6043).
+
+## Drawing
+
+After the current view of a layer's data has been sliced into RAM, the data is pushed to VRAM with any associated transforms and parameters.
+
+Each napari layer type has a corresponding vispy layer type.
+For example, the [`VispyImageLayer`](https://github.com/napari/napari/blob/5e8dc098cb213c5f963524e619f223ad4fe90be8/napari/_vispy/layers/base.py#L21)
+corresponds to the {class}`Image<napari.layers.Image>` layer.
+Each instance of a layer also has a corresponding instance of its vispy layer.
+These correspondences can be found [`VispyCanvas.layer_to_visual`](https://github.com/napari/napari/blob/5e8dc098cb213c5f963524e619f223ad4fe90be8/napari/_vispy/canvas.py#L69).
+
+The vispy layer instance has a reference to its corresponding layer.
```

Comment:
Above you say 

> Each instance of a layer also has a corresponding instance of its vispy layer.

So does that mean that the layer instance and vispy layer instance both hold references to each other?

Reply:
Yeah maybe this should be worded differently. Vispy side holds references to the models and uses events to get updates, but the models know nothing of the vispy side.

Reply:
I reworded

> Each instance of a layer also has a corresponding instance of its vispy layer.

to

> For each instance of a layer, there is a corresponding instance of its type's vispy layer

so that it doesn't sound like the layer has/owns the vispy layer.